resources/hab_build.rb

```diff
@@ -39,7 +63,10 @@
 
 action :publish do
   execute 'upload-pkg' do
-    command lazy { "#{hab_binary} pkg upload --url #{url} #{hab_studio_path}/src/results/#{artifact}" }
+    command(lazy do
+      url_opt = "--url #{url}" if url
```

Comment:
Can we rename the `url` property to `depot_url` also? `url` is too general of a property name.